android-client-sdk/src/main/java/com/devcycle/sdk/android/api/DVCClient.kt

```diff
@@ -448,6 +451,11 @@ class DVCClient private constructor(
             this.context = context
             return this
         }
+
+        fun withHandler(handler: Handler): DVCClientBuilder {
```

Comment:
any way to make this one only accessible to tests so that it doesnt show up in the public interface?

Reply:
yes